packages/core/src/text/StringUtils.ts

```diff
@@ -1,7 +1,6 @@
 import { EMPTY_STRING, SPACE } from './Strings';
 import { RegExpUtils } from './RegExpUtils';
-import { CollectionUtils } from '../collections/base/CollectionUtils';
-import { ReadOnlyCollection } from '../collections/collection/readonly/ReadOnlyCollection';
+import { CollectionUtils, ReadOnlyCollection } from '@monument/collections';
```

Comment:
Seems like `text` package is next candidate for extraction ;)

Reply:
for sure it is:+1: